lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -15322,7 +15322,7 @@ msgid "Host group"
 msgstr "Groupe d'hôtes"
 
 msgid "Downtime"
-msgstr "Plage de maintenance"
+msgstr "Planifier une plage de maintenance"
```

Comment:
This isn't the correct translation, the previous one was actually accurate.

"Planifier une plage de maintenance" should translate to "Set downtime" in english

Reply:
There are mistakes on each dev / stable branches on 21.10, 21.04 and 20.10. It is a bit messy, I think that we should close this PR, check translation on dev and stable branches and, if needed, create a PR for each version